uahf-technical-spec.md

```diff
@@ -15,9 +15,8 @@ fork - the sense of the requirements would largely stay the same.
 
 ## Definitions
 
-MTP: the "median time past" value of a block, calculated from the
-nTime values of its past up to 11 ancestors, as obtained by the
-GetMedianTimePast(block.parent) call.
+MTP: the "median time past" value of a block, calculated from its nTime
+value, and the nTime values of its up to 10 immediate ancestors.
```

Comment:
As this is a spec, you may want to actually define the calculation:

In then takes those 11 values, sorts them and returns the middle one.